lms/djangoapps/discussion_api/tests/test_views.py

```diff
@@ -711,6 +711,7 @@ def test_basic(self):
             "username": self.user.username,
             "created_at": "2015-05-19T00:00:00Z",
             "updated_at": "2015-05-19T00:00:00Z",
+            "read": True,
```

Comment:
On posting a thread, response and/or comment; the thread should be marked as `read` for the owner user. Here we are mocking the response of `cs_comment_service` that returns `read=true` on newly created thread object.